testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -140,30 +146,34 @@ def function(x):
 
         arguments = [tuple([i]) for i in np.arange(0,100)]
 
-        parallel_calculation = encore.utils.ParallelCalculation(function = function,
-                                                                ncores = 4,
-                                                                args = arguments)
+        parallel_calculation = encore.utils.ParallelCalculation(function=function,
+                                                                n_jobs=4,
+                                                                args=arguments)
         results = parallel_calculation.run()
 
         for i,r in enumerate(results):
             assert_equal(r[1], arguments[i][0]**2,
                 err_msg="Unexpeted results from ParallelCalculation")
 
-    def test_rmsd_matrix_with_superimposition(self):        
+    @dec.skipif(module_not_found('sklearn'),
+                "Test skipped because sklearn is not available.")
+    def test_rmsd_matrix_with_superimposition(self):
```

Comment:
I see. Thanks. I thought maybe sklearn would expose joblist as a global module when importing sklearn - although now that I think about it that would be a pretty ugly side effect. 

I don't want to turn this into a huge deal. I can live with it either way - and you certainly have a better idea of what the overall strategy is for MDAnalysis. So, your call.